lib/rules/require-lang-attribute.js

```diff
@@ -1,7 +1,6 @@
-import tags from 'language-tags';
```

Comment:
ah I see -- why don't we want to leave this as is? is it not maintained? :see_no_evil:

Reply:
It's adding 800kb of `json` to bundle (